### PR TITLE
YDA-5946 - Modified form group with Yoda buttons to stick to the top of metadata form.

### PR DESCRIPTION
**Pull request by kaur16**
YDA-5946 - Modified form group with Yoda buttons to stick to the top of metadata form.

### CODE REVIEW

**Comment by kaur16** [comment]
Let me know what you think about this!

File: research/metadata-form/src/index.js [hunk 1]
```diff
@@ -521,43 +521,66 @@ class YodaButtons extends React.Component {
...
+        return (<button onClick={this.props.cloneMetadata} type="button" className="btn btn-primary clone-metadata-btn me-3" title="Clone from parent folder">Clone</button>);
     }
 
     renderFormCompleteness() {
-        return (<div><span className="text-sm float-start text-muted text-center ms-3 mt-1">Required for the vault:</span><div className="form-completeness progress float-start ms-3 mt-2 w-25" data-bs-toggle="tooltip" title=""><div className="progress-bar bg-success"></div></div></div>);
+        return (<div><span className="text-sm float-start text-muted text-center ms-3 mt-1">Required for Vault:</span><div className="form-completeness progress float-start ms-3 mt-2 w-25" data-bs-toggle="tooltip" title=""><div className="progress-bar bg-success"></div></div></div>);
+    }
+
+    renderCloseButton() {
+        return(<a class="btn btn-secondary" href={"/research/browse?dir=" + encodeURIComponent(path)}>Close</a>);
```
Comment (lwesterhof) [inline research/metadata-form/src/index.js]: Can we also add a title to the close button?

File: research/metadata-form/src/index.js [hunk 2]
```diff
@@ -606,9 +629,6 @@ class Container extends React.Component {
                              deleteMetadata={deleteMetadata}
                              cloneMetadata={this.cloneMetadata} />
                 <YodaForm ref={(form) => {this.form=form;}}/>
-                <YodaButtons saveMetadata={this.saveMetadata}
-                             deleteMetadata={deleteMetadata}
-                             cloneMetadata={this.cloneMetadata} />
             </div>
```
Comment (lwesterhof) [inline research/metadata-form/src/index.js:634]: We can pull the card header and card body into this render statement. 

Then it should look something like this:
Screenshot 2024-10-21 at 08-49-48 Yoda - Dev - Research

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: research/metadata-form/src/index.js [hunk 3]
```diff
@@ -733,7 +755,8 @@ function loadForm() {
 
                 // If maintenance banner is visible, add padding to metadata form header
```
Comment (lwesterhof) [inline research/metadata-form/src/index.js:756]: Can we do the same for the "This is a development/testing/acceptance environment!" banner?

**Review by lwesterhof — CHANGES_REQUESTED** [review]

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**